module-services/service-desktop/endpoints/reboot/RebootEndpoint.cpp

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) 2017-2023, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include <endpoints/reboot/RebootEndpoint.hpp>
+#include <endpoints/HttpEnums.hpp>
+#include <log/log.hpp>
+#include <module-sys/SystemManager/include/SystemManager/SystemManagerCommon.hpp>
+#include <endpoints/message/Sender.hpp>
+
+namespace sdesktop::endpoints
+{
+    auto RebootEndpoint::handle(Context &context) -> void
+    {
+        switch (context.getMethod()) {
+        case http::Method::post: {
+
+            const auto &body = context.getBody();
```

Comment:
You can create a file with helper functions and move it there.
You can check how it is done in other helpers
function name: handlePost(context)